docs/examples/example_satellite_observations.ipynb

```diff
@@ -0,0 +1,360 @@
+{
```

Comment:
<p>This cell can be written in a one-liner: </p><p> </p><pre class="ql-syntax" spellcheck="false">wetMask = xr.where(np.isnan(ke_og), 0, 1)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocean-eddy-cpt/gcm-filters/pull/92/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>